.circleci/config.yml

```diff
@@ -0,0 +1,168 @@
+version: 2
+jobs:
+  install-dependencies:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+      - run: yarn
+      - run: echo "Checking if lockfiles changed..." && git diff --exit-code
+      - save_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+      - run: npm rebuild node-sass
+
+  compile-libs:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+      - run: yarn compile-libs
+      - persist_to_workspace:
+          root: '.'
+          paths:
+            - yarn.lock
+            - packages/core/dist
+            - packages/datetime/dist
+            - packages/docs/dist
+            - packages/labs/dist
+            - packages/table/dist
+
+  lint:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - attach_workspace:
+          at: '.'
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+      - run: yarn lint
+
+  dist:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - attach_workspace:
+          at: '.'
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+      - run: yarn dist
+      - persist_to_workspace:
+          root: '.'
+          paths:
+            - yarn.lock
+            - packages/core/dist
+            - packages/datetime/dist
+            - packages/docs/dist
+            - packages/labs/dist
+            - packages/table/dist
+
+  test-core:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - attach_workspace:
+          at: '.'
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+      - run: ./node_modules/.bin/lerna run test --scope '@blueprintjs/core'
+
+  test-datetime:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - attach_workspace:
+          at: '.'
+      - restore_cache:
+          key: v1-dependency-cache-{{ checksum "yarn.lock" }}
+      - run: ./node_modules/.bin/lerna run test --scope '@blueprintjs/datetime'
```

Comment:
`yarn lerna run test ...args`?

`yarn <binary>` is equivalent to `./node_modules/.bin/<binary>`

Reply:
yeah makes sense